packages/server/graphql/mutations/helpers/notifications/getSummaryText.ts

```diff
@@ -1,11 +1,12 @@
 import plural from 'parabol-client/utils/plural'
 import relativeDate from 'parabol-client/utils/date/relativeDate'
-import MeetingRetrospective from '../../../../database/types/MeetingRetrospective'
-import MeetingAction from '../../../../database/types/MeetingAction'
-import MeetingPoker from '../../../../database/types/MeetingPoker'
+import {isMeetingRetrospective} from '../../../../database/types/MeetingRetrospective'
+import {isMeetingAction} from '../../../../database/types/MeetingAction'
+import {isMeetingPoker} from '../../../../database/types/MeetingPoker'
+import Meeting from '../../../../database/types/Meeting'
 
-const getSummaryText = (meeting: MeetingRetrospective | MeetingAction | MeetingPoker) => {
-  if (meeting.meetingType === 'retrospective') {
+const getSummaryText = (meeting: Meeting) => {
+  if (isMeetingRetrospective(meeting)) {
```

Comment:
Typecheck functions were added to avoid casting

Reply:
Yes, that's right. Before this worked because the meeting was handed down directly with the correct type. If I have an `endMeeting(meetingId)` function, then at some point I would need to check if it matches any of those types. This is the place now where this is done, when we need this distinction.

Reply:
I'm not sure I follow. Do you have a code example of the benefit?

Reply:
It's easy enough to do it afterwards. Dataloader is still kept out of the implementations, so resolving for now as it's not a pressing issue.